File: CarServiceGame/CarServiceGame.Domain/Entities/GarageRanking.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CarServiceGame.Domain.Entities
{
    public class GarageRanking
    {
        public Guid GarageId { get; set; }
        public string Name { get; set; }
        public decimal CashBalance { get; set; }
        public int NumberOfWorkers { get; set; }
        public int NumberOfCompletedOrders { get; set; }
        public int Efficiency { get; set; }

        public GarageRanking()
        {

        }

        public GarageRanking(Guid garageId, string name, decimal cashBalance, int numberOfWorkers, int numberOfCompletedOrders, int efficiency)
        {
            GarageId = garageId;
            Name = name;
            CashBalance = cashBalance;
            NumberOfWorkers = numberOfWorkers;
            NumberOfCompletedOrders = numberOfCompletedOrders;
            Efficiency = efficiency;
        }

    }
}

File: CarServiceGameWeb/src/app/model/garage-ranking.ts
export class GarageRanking {
    GarageName: string;
    Balance: number;
    NumberOfWorkers: number;
    NumberOfCompletedOrders: number;
    Efficiency: number;

    constructor(name, balance, noOfWorkers, noOfCompletedOrders, efficiency) {
        this.GarageName = name;
        this.Balance = balance;
        this.NumberOfWorkers = noOfWorkers;
        this.NumberOfCompletedOrders = noOfCompletedOrders;
        this.Efficiency = efficiency;
    }
}